pkgs/development/libraries/mbedtls/default.nix

```diff
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
     sha256 = "042q1l4708zjn5v72sa9qdvgx173kmy4hbcd23wj5vqd6vbmk6d9";
   };
 
-  nativeBuildInputs = [ perl ];
+  nativeBuildInputs = [ perl ] ++ stdenv.lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
 
   patchPhase = stdenv.lib.optionalString stdenv.isDarwin ''
     substituteInPlace library/Makefile --replace "-soname" "-install_name"
```

Comment:
fixing the `install_name` here to use the absolute store path should allow us to get rid of the postInstall patching.